examples/app.jsx

```diff
@@ -49,6 +49,7 @@ class TwoSquares extends React.Component {
       <div>
         <div key="one" style={[squareStyles.both, squareStyles.one]} />
         <div key="two" style={[squareStyles.both, squareStyles.two]} />
+        <div key="three" disabled style={[squareStyles.both, squareStyles.three]} />
```

Comment:
On line 177 and up here, let's call that style `all` instead of `both` since there are three now.